src/ui-test/xml_completion_support_test.ts

```diff
@@ -225,9 +225,8 @@ describe('XML DSL support', function () {
 	 *
 	 * @param item ContenAssistItem
 	 */
-	async function getTextExt(item: ContentAssistItem): Promise<String> {
-		let name: string = '';
-		name = await item.getText();
+	async function getTextExt(item: ContentAssistItem): Promise<string> {
+		let name: string = await item.getText();
```

Comment:
this can be even made a `const`

Reply:
https://github.com/camel-tooling/camel-lsp-client-vscode/pull/493